packages/ra-data-simple-prisma/src/permissions/canAccess.ts

```diff
@@ -32,12 +32,15 @@ export const canAccess = ({
   field,
 }: {
   action: Action;
-  permissions: Permissions<string>;
-  resource: string; //better type is Permission["resource"] but ra's resource is string
+  permissions: Permissions<any>;
```

Comment:
this should be any, string will be clash with someone's more specific table key definition. The Object is still mostly typed.